src/util.ts

```diff
@@ -393,3 +393,27 @@ const addGraphTotals = (
   )
   return { totalTxs, totalUsd }
 }
+
+export const movingAveDataSort = (
+  data: Data[]
+): Array<{ date: string; allTxs: number }> => {
+  const aveArray: Array<{ date: string; allTxs: number }> = []
+  if (Array.isArray(data) === false) return aveArray
+  const dataLength = data.length
+  let sevenDaySum: number = 0
+  for (let i = 0; i < dataLength; i++) {
+    for (let j = i; j >= i - 6; j--) {
```

Comment:
The `6` feels like a magic number. But I'm assuming it's for the number of days. Perhaps replace it with a variable that's a little clearer.

Reply:
Changed according to comment.